src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2312,11 +2320,19 @@ protected virtual LocalFunctionSymbol LookupLocalFunction(SyntaxToken nameToken)
 
         internal BoundBlock BindEmbeddedBlock(BlockSyntax node, DiagnosticBag diagnostics)
         {
-            return this.GetBinder(node).BindBlock(node, diagnostics);
+            return BindBlock(node, diagnostics);
```

Comment:
You've moved the effect of BindEmbeddedBlock into the method it now calls. I suspect there is no longer a need for BindEmbeddedBlock.